notebooks/02_analyses/hatesonar_analysis.ipynb

```diff
@@ -0,0 +1,1423 @@
+{
```

Comment:
<p>df @cdolfi can you save the processed results dataframe <code>clean</code> to csv here instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>